src/reducers/party.js

```diff
@@ -0,0 +1,12 @@
+import types from '../actions';
+
+const party = (state = {}, action) => {
+  switch (action.type) {
+    case types.party.set:
+      return Object.assign({}, state, action.party);
```

Comment:
As `setParty` is meant to replace the current party you could just return `action.party`